namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-restricted-patients-api-dev/resources/variables.tf

```diff
@@ -0,0 +1,36 @@
+variable "domain" {
+  default = "restricted-patients-api-dev.service.justice.gov.uk"
```

Comment:
Can you amend the domain name please, 
https://ministryofjustice.github.io/technical-guidance/documentation/standards/naming-domains.html#naming-domains